skins/index.js

```diff
@@ -336,6 +337,7 @@ export default {
   bee2,
   benchmark,
   betswirl,
+  breederdaoSpace,
```

Comment:
When you use a dash, you must use this naming convention.
```suggestion
  "breederdao-space": breederdaoSpace,
```

Reply:
> When you use a dash, you must use this naming convention.

resolved this issue @gokaiorg thanks!